scripts/run-cni-release-tests.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+# script to run integration and calico tests tests(no cluster creation & deletion/installing addons)
+# use case: run script after CNI images are updated to the image to be verified (see update-cni-images.sh)
+
+# CLUSTER_NAME: name of the cluster to run the test
+# VPC_ID: cluster VPC ID
+# REGION: default us-west-2
+# KUBE_CONFIG_PATH: path to the kubeconfig file, default ~/.kube/config
+# NG_LABEL_KEY: nodegroup label key, default "kubernetes.io/os"
+# NG_LABEL_VAL: nodegroup label val, default "linux"
+
+# CALICO_VERSION: calico version, default 3.22.0
+
+set -e
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+TEST_DIR="$SCRIPT_DIR/../test"
+INTEGRATION_TEST_DIR="$TEST_DIR/integration-new"
+CALICO_TEST_DIR="$TEST_DIR/e2e/calico"
+
+source "$SCRIPT_DIR"/lib/cluster.sh
+source "$SCRIPT_DIR"/lib/integration.sh
+
+function run_integration_test() {
```

Comment:
Will there be another script for ipv6 and cni-metrics-helper?

Reply:
I can add cni-metrics-helper tests in the same script. 
Will take a look if ipv6 tests needs another script.